MAUI/LinearGauge/bar-pointer.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: post
+title: Bar Pointer in .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here all about adding and customizing Bar Pointer of Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control and more.
+platform: maui
+control: SfLinearGauge
+documentation: ug
+---
+
+# Bar Pointer in .NET MAUI Linear Gauge (SfLinearGauge)
+
+A bar pointer is an accenting line or shaded background that can be placed on a Linear Gauge to mark any current value in the axis track. The bar pointers always start from the minimum value of the axis and end with the specified value. So the [`Value`]() property is a required parameter for creating a bar pointer.
+
+## Default bar pointer
+
+The following code sample creates a default bar pointer with the value 50. 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<gauge:SfLinearGauge>
+                <gauge:SfLinearGauge.BarPointers>
+                    <gauge:BarPointer Value="50"/>
+                </gauge:SfLinearGauge.BarPointers>
+            </gauge:SfLinearGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		gauge.BarPointers.Add(new BarPointer()
+		{
+			Value = 50,
+		});
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Initialize linear gauge for bar pointer](images/bar-pointer/default_bar_pointer.PNG)
+
+## Customize bar pointer thickness
+
+The thickness can be changed by the [`PointerSize`]() property of the bar pointer. The following code sample demonstrates the same. 
```

Comment:
The thickness can be changed using the [`PointerSize`]() property of the bar pointer.